src/dodal/devices/athena/panda/tables.py

```diff
@@ -0,0 +1,232 @@
+from collections import deque, namedtuple
+from itertools import cycle
+
+import numpy as np
+from ophyd_async.core import observe_value
+from ophyd_async.panda import SeqTable
+from pyfiglet import Figlet
```

Comment:
seems like this library was only used for a demo peter worked on, and everything relating to it could be removed.